ush/config_defaults.yaml

```diff
@@ -1042,43 +1029,34 @@ nco:
   #
   #-----------------------------------------------------------------------
   #
-  envir: "para"
-  NET: "rrfs"
-  RUN: "rrfs"
-  model_ver: "v1.0.0"
+  envir_dfv: "para"
+  NET_dfv: "rrfs"
+  RUN_dfv: "rrfs"
+  model_ver_dfv: "v1.0.0"
```

Comment:
If we don't use `ecFlow`, you are correct because Rocoto does not use job cards and these NCO variables don't have to be replaced with the new values in the job cards. However, as you can see in Table 1 (pp.4) of the NCO standards, the NCO variables should be defined in the job cards. When these NCO variables are defined in the job cards, they should replace these variables ({var}_dfv) defined in config.yaml and ush/machine/.

Reply:
Can that requirement not be interpreted as sourcing the `var_defns.sh` file in the job card? It doesn't explicitly state that the variables are written out there and set. 

I also am worried that this requirement leads to unnecessary duplication of information and general misdirection. For example, where do we point users for setting these values? Typically it's been a one-stop shop for controlling this information in the config_defaults.sh. To override those defaults, putting the information in the config.yaml is the one way to specify a different value.

Even if we do need to adopt this approach for this handful of variables, we need to be consistent with defining them in Rocoto and in ecFlow as environment variables.

It's unclear on whether the requirement here is meant to reflect that _environment_ has these variables set in the job card or if the job card itself is responsible for setting the single source of information to be used for all other scripts. Can you speak to that distinction?